src/main/scala/cromwell/binding/types/WdlArrayType.scala

```diff
@@ -27,3 +27,14 @@ case class WdlArrayType(memberType: WdlType) extends WdlType {
     case _ => false
   }
 }
+
+object WdlArrayType {
+
+  implicit class WdlArrayEnhanced(wdlType: WdlType) extends WdlType {
+
+    override protected def coercion: PartialFunction[Any, WdlValue] = wdlType.coercion
```

Comment:
My scala's not great on implicits... I don't see how this enhances the basic coercion behaviour?